R/score.R

```diff
@@ -0,0 +1,57 @@
+# score.R
+
+#' Produce a gH object: a hash of genes and their associated scores
+#' (likelihood that observed mutations exceed background mutations at FDR of 1%)
+#'
+
+score <- function(gX, gH, silent = FALSE, noLog = FALSE) {

```

Comment:
The common part of all our function signatures is ```writeLog = TRUE```. I have even commented on that change explicitly on Riot.